### PR TITLE
DSD-1221: Add onChangeEnd prop to Slider

### PR DESCRIPTION
**Pull request by jackiequach**
Fixes JIRA ticket [DSD-1221](https://jira.nypl.org/browse/DSD-1221)

## This PR does the following:

- Adds the `onChangeEnd` prop in the `Slider` component.

## How has this been tested?

Locally in StoryBook and unit tests.

## Accessibility concerns or updates

-

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/components/Slider/Slider.test.tsx [hunk 1]
```diff
@@ -374,6 +374,62 @@ describe("Slider", () => {
...
+      });
+      expect(currentValue).toEqual(42);
+
+      fireEvent.change(input, {
+        target: { value: "84" },
+      });
+      expect(currentValue).toEqual(84);
+    });
+
+    it("logs a warning when there is both `onChange` and `onChangeEnd` are passed", () => {
```
Comment (EdwinGuzman) [inline src/components/Slider/Slider.test.tsx]: Nitpick: you don't need to include "there is" in the sentence.

**Review by EdwinGuzman — COMMENTED** [review]
This is good. Can you add an example and a test for when it's used in the `isRangeSlider` variant?

**Review by EdwinGuzman — APPROVED** [review]
Good update!

**Review by bigfishdesign13 — APPROVED** [review]
This is great.

**Comment by bigfishdesign13** [comment]
@jackiequach This is good to go.  If you merge it in today or early tomorrow, we can include it in the `1.4.0` release.

**Comment by jackiequach** [comment]
@bigfishdesign13 Should I update the version in the StoryBook page for these changes?

**Comment by bigfishdesign13** [comment]
> @bigfishdesign13 Should I update the version in the StoryBook page for these changes?

@jackiequach Yes, good point.  It would need the `Latest Version` number updated to `1.4.0`.  Edwin and I both missed that when we approved the PR.  Oops.

**Pull request merged**